uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -186,9 +186,28 @@ public static function getAvailableColors() {
    * @return array An array containing the values
    */
   public static function getValuesArray($id) {
-    $result = civicrm_api3('AbsenceType', 'get', array('id' => $id));
+    $result = civicrm_api3('AbsenceType', 'get', ['id' => $id]);
```

Comment:
No reason to be honest. I believe that, when I first created it, I based myself in some other form that was using the same strategy.

Since this is being used only by the form, I think it's fine to leave it as it is.